dotcom-rendering/src/components/GalleryImage.tsx

```diff
@@ -0,0 +1,94 @@
+import { css } from '@emotion/react';
+import { from, space, until } from '@guardian/source/foundations';
+import { grid } from '../grid';
+import { type ArticleFormat } from '../lib/articleFormat';
+import { getImage } from '../lib/image';
+import { palette } from '../palette';
+import { type ImageBlockElement } from '../types/content';
+import { GalleryCaption } from './GalleryCaption';
+import { Picture } from './Picture';
+
+type Props = {
+	format: ArticleFormat;
+	image: ImageBlockElement;
+	pageId: string;
+	webTitle: string;
+};
+
+const styles = css`
+	${grid.container}
+	grid-auto-flow: row dense;
+	column-gap: ${space[5]}px;
```

Comment:
This is `20px`. Given that `grid.container` already sets this property to `20px` for all breakpoints except `mobile`, I think the net effect of this line is to increase the column gap from `10px` to `20px` at mobile?

The consequence of this change seems to be that we get `20px` of padding either side of the image captions at `mobile`? But the existing galleries on dotcom only have `10px` padding here, so perhaps we should remove this and use the `column-gap` set by `grid.container`?